asset/css/prefix-button.less

```diff
@@ -0,0 +1,9 @@
+.submit-btn-duplicate {
```

Comment:
I would call it `primary-submit-btn-duplicate`. And please fix the filename as well.